internal/controller/eventing/integrationtests/controller/integration_test.go

```diff
@@ -132,11 +132,13 @@ func Test_CreateEventingCR(t *testing.T) {
 
 			testEnvironment.EnsureNamespaceCreation(t, givenNamespace)
 			// when
-			testEnvironment.EnsureK8sResourceCreated(t, tc.givenEventing)
+			// create NATS.
 			testEnvironment.EnsureK8sResourceCreated(t, tc.givenNATS)
```

Comment:
For reviewer: First creating NATS CR before Eventing CR, so the reconciler do not give NATS not available error.